primary/locales/en.json

```diff
@@ -419,12 +414,39 @@
 			"setCurrentEventKey": "Set current event key",
 			"successClear": "Cleared current event successfully.",
 			"successSetEvent": "Set current event to *{eventName}* successfuly."
+		},
+		"assignments": {
+			"pit": "Pit scouting assignments",
+			"match": "Match scouting assignments",
+			"assignPitTeams": "Assign teams to pit scouting groups",
+			"assignMatchTeams": "Assign matches to selected scouters",
+			"availablePotentialScouters": {
+				"main": "Available potential scouters",
+				"tooltip": "The scouters and members that appear in this list are only those who are [marked as present](/manage/members/present).",
+				"noneFound": "No members were found. You must [mark some as present](/manage/members/present)."
+			},
+			"blockSize": {
+				"main": "Allocate in blocks of...",
+				"1": "If you have only 6 assignees then they’ll be doing every match with no breaks.",
+				"2": "If you have 12 assignees total, they’ll be doing N matches on, then N matches off, loop. Or if you have more than 12 assignees, they might occasionally get 2xN matches off. A suggested “N” is 5.",
+				"3": "If you have 7 to 11 assignees, they will occasionally do 2xN matches on before getting N matches off. A suggested “N” is 3."
```

Comment:
I opted for the former, labeling them `tooltip_line1`, etc., because I feel like it would be cumbersome for our translators to be careful about the exact Markdown syntax.

Reply:
I'm not really a fan of just indexing them- maybe name them like something like `tooltipAssigned6` or `tooltipAssigned.6` to represent their actual contents.

I don't think having translators do markdown is that difficult though, Weblate does have consistency checking for it so I'm pretty sure it would warn if the list is formatted wrong.

Reply:
@caburum It seems like marked.parseInline doesn't include lists, so I don't think it is possible to do it in one string without modifying i18n.ts.

Tried with this:
`" - If you have only 6 assignees then they’ll be doing every match with no breaks. \n - If you have 12 assignees total, they’ll be doing N matches on, then N matches off, loop. Or if you have more than 12 assignees, they might occasionally get 2xN matches off. A suggested “N” is 5. \n - If you have 7 to 11 assignees, they will occasionally do 2xN matches on before getting N matches off. A suggested “N” is 3."`

and it results in this HTML:

```html
<span class="w3-tooltiptext w3-left-align"> - If you have only 6 assignees then they’ll be doing every match with no breaks.<br> - If you have 12 assignees total, they’ll be doing N matches on, then N matches off, loop. Or if you have more than 12 assignees, they might occasionally get 2xN matches off. A suggested “N” is 5.<br> - If you have 7 to 11 assignees, they will occasionally do 2xN matches on before getting N matches off. A suggested “N” is 3.</span>
```